include/EVT/io/GPIO.hpp

```diff
@@ -63,7 +72,7 @@ class GPIO {
      * @param[in] pin The pin for the GPIO instance to use.
      * @param[in] direction The directional flow of data.
      */
-    GPIO(Pin pin, Direction direction);
+    GPIO(Pin pin, Direction direction, Pull pull = Pull::PULL_DOWN);
```

Comment:
In the summary of this PR, you say that the pull is supposed to default to NO_PULL, but here you have it as PULL_DOWN. Is this correct?

Reply:
Good catch, should be `PULL_DOWN`